test/suite/test_tiered12.py

```diff
@@ -91,13 +91,6 @@ def test_tiered(self):
 
         # Sleep more than the one second stress timing amount and give the thread time to run.
         time.sleep(2)
-        # After sleeping, the internal thread should have created the cached object.
-        if self.has_cache:
```

Comment:
Also the comment above line 70 needs to be edited. 

>         # So after flush tier completes, check that the cached object does not
>         # exist. Then sleep and check that it does exist.
>         #
>

Reply:
removed [12ff933](https://github.com/wiredtiger/wiredtiger/pull/9708/commits/12ff9339c1e48e451f1f053627295e0c4ddb4836)